vyper/compiler/output_bundle.py

```diff
@@ -160,6 +166,10 @@ def write(self):
         self.write_settings(self.compiler_data.original_settings)
         self.write_integrity(self.compiler_data.resolved_imports.integrity_sum)
         self.write_sources(self.bundle.compiler_inputs)
+        if self.compiler_data.storage_layout_override:
```

Comment:
don't use truthy unless you mean it!
```suggestion
        if self.compiler_data.storage_layout_override is not None:
```